backends/p4tools/modules/testgen/core/small_step/table_stepper.h

```diff
@@ -73,7 +73,8 @@ class TableStepper {
 
     /// Sets the action taken by the given table. The arguments in the given MethodCallExpression
     /// are assumed to be symbolic values.
-    const IR::StringLiteral *getTableActionString(const IR::MethodCallExpression *actionCall);
+    static const IR::StringLiteral *getTableActionString(
+        const IR::MethodCallExpression *actionCall);
```

Comment:
This function is defined in small_step/table_stepper.cpp (and used there) and also used in targets/pna/shared_table_stepper.cpp (and not defined there).  So I'm suprised it still compiles if you make the declaration `static` -- does it depend on unified build pulling them into the same compilation unit?

Reply:
This is a static class member function so the derived classes should be able to access it? This also works without a unified build.

Reply:
Ah, my bad -- I thought it was a file-scopt function, not a member.